.gitignore

```diff
@@ -2,4 +2,7 @@
 *~
 handwritting/*
 dataset/*
-credentials.json
\ No newline at end of file
+credentials.json
+logs/*
+nohup.out
+model_checkpoint/*
```

Comment:
Can't comment on the notebook as it is too large, will comment here:
`paragraph_segmentation_dcnn.ipynb`
=> make the training loop, evaluation loop dependent on a `run_epoch(net, dataloader, loss_function)` function and simplify your notebook by using it in each place instead of creating a new loop.

Reply:
`paragraph_segmentation_msers.ipynb` => good research notebook, though could have a bit more explanation than "doesn't generalize well" where all the above examples are example of perfect segmentation. One or two failure case would be nice to elaborate on that.